pkg/plugin/sdk/v1alpha2/plugin/interface.go

```diff
@@ -62,3 +73,42 @@ func (*Base) PostTestHook(bool, ioutils.PromptRunner) error {
 func (*Base) PostRunHook(bool, ioutils.PromptRunner) error {
 	return nil
 }
+
+type CustomCommandFn (func(ctx context.Context, args []string, bzl bazel.Bazel) error)
+
+type Command struct {
+	Use       string
+	ShortDesc string
+	LongDesc  string
+	Run       CustomCommandFn
+}
+
+type CommandManager interface {
+	Save(commands []*Command) error
+	Execute(command string, ctx context.Context, args []string) error // Not sure about this signature, haven't read the whole code yet.
```

Comment:
Lets make sure we address or delete the comment here...

Reply:
Removing. Ive tested and the signature works